generators/frontend/templates/react-demo/pages/_document.js

```diff
@@ -0,0 +1,28 @@
+import Document, { Head, Main, NextScript } from 'next/document'
+
+export default class MyDocument extends Document {
+  static async getInitialProps(ctx) {
+    const initialProps = await Document.getInitialProps(ctx)
+    return { ...initialProps }
+  }
+  render() {
+    return (
+      <html>
+        <Head>
+          <meta name='viewport' content='initial-scale=1.0, width=device-width' />
+          <style>{`body { margin: 0 } /* custom! */`}</style>
+          <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js" defer></script>
+          <script src="https://cdn.timekit.io/booking-js/v2/booking.min.js" defer></script>
```

Comment:
Instead of the scripts here you can run ```npm install jquery fullcalendar```
Run the command, comment the lines related to fullcalendar and see if everything works as expected